build/journalbeat/config/journalbeat.yml

```diff
@@ -0,0 +1,11 @@
+journalbeat.inputs:
+- paths: [/mnt/journal]
```

Comment:
I used [this config](https://github.com/elastic/beats/blob/9294384563a3f7ecd1ba9b532948279f0b528952/journalbeat/_meta/beat.docker.yml) in new docker images:
```
journalbeat.inputs:
- paths: []
  seek: tail
```

Not sure what option would be better but we should keep them synced.

Reply:
The default value is `cursor` for `seek`. I think we should stick to that.

Reply:
You are right, I have opened a PR for that #9233

What about the paths config?